ckanfunctionaltests/api/test_organizations.py

```diff
@@ -130,9 +130,9 @@ def test_organization_show_inc_datasets_stable_pkg(
     with subtests.test("response validity"):
         validate_against_schema(rj, "organization_show")
 
-    desired_result = tuple(
-        pkg for pkg in rj["result"]["packages"] if pkg["organization"]["name"] == stable_org_with_datasets["name"]
-    )
+    desired_result = [
```

Comment:
This shouldn't be called `desired_result` anymore seeing as it's looking to pick out more than one item. The idea was that here we were plucking out of all the results the *one* we were interested in to do the comparison against. Perhaps `interesting_results` or something, dunno.

Reply:
I think I'll just leave this for now